ts/main.ts

```diff
@@ -0,0 +1,64 @@
+// interface for Monster
+interface Monster {
+  id: number;
+  name: string;
+  type: string;
+  description: string;
+  locations: [];
+}
+
+const $row = document.querySelector('.row');
+
+async function makeMonsters(): Promise<void> {
+  if (!$row) throw new Error('Could not load row');
+  for (let i = 1; i < 59; i++) {
+    try {
+      const response = await fetch(`https://mhw-db.com/monsters/${i}`);
+      if (!response.ok) {
+        throw new Error('This is not ok');
+      }
+      const monster = (await response.json()) as Monster;
+      console.log(monster);
+      const newPage = renderMonster(monster);
+      $row.append(newPage);
+    } catch (error) {
+      console.log('Error:', error);
+    }
```

Comment:
I would change this to `console.error` rather than `console.log`. It will look the same for this project but generally with an error you would want it to print for yourself and not the users.